core/src/main/java/org/pentaho/di/shared/VfsSharedObjectsIO.java

```diff
@@ -186,18 +186,15 @@ private static String  getDefaultSharedObjectFileLocation() {
    * @throws KettleException
    */
   public void saveSharedObject( String type, String name, Node node ) throws IOException, KettleException {
-    // Find the map for the type
+    // Get the map for the type
     Map<String, Node> nodeMap = getNodesMapForType( type );
-    if ( nodeMap.containsKey( name ) ) {
-      // Update the map entry for this name
-      nodeMap.put( name, node );
-
-      FileObject fileObject = KettleVFS.getFileObject( sharedObjectFile );
-      Optional<String> backupFileName = createOrGetFileBackup( fileObject );
-      writeToFile( fileObject, backupFileName );
-      isInitialized = false;
-    }
-    // TODO need to handle the case when the new shared object is being added
+    // Add or Update the map entry for this name
+    nodeMap.put( name, node );
+
+    FileObject fileObject = KettleVFS.getFileObject( sharedObjectFile );
```

Comment:
I would add a comment about this not needing a bowl

Reply:
Not sure what kind of comment you like to see added

Reply:
Was thinking something like "Does not need a bowl because the path to a project can't use a project VFS connection"

Reply:
but really not a big deal.